app/plugins/acl.md

```diff
@@ -62,7 +68,7 @@ You can have more than one group associated to a consumer.
 
 ### Upstream Headers
 
-When a consumer has been validated, the plugin will append a `X-Consumer-Groups` header to the request before proxying it to the upstream service, so that you can identify the groups associated with the consumer. The value of the header is a comma separated list of groups that belong to the consumer, like `admin, pro_user`.
+When a consumer has been validated, the plugin will append a `X-Consumer-Groups` header to the request by default before proxying it to the upstream service, so that you can identify the groups associated with the consumer. The value of the header is a comma separated list of groups that belong to the consumer, like `admin, pro_user`. This header can be hidden by setting the `hide_groups_header` flag to true.
```

Comment:
Can we replace with:
```
pro_user.

This header will not be injected in the request to the upstream service if the `hide_groups_header` config flag is set to `true`.
```

(Note the line jump). We can also get rid of the `by default` addition in the first sentence which I think is more confusing to read.

Reply:
Agree, that is cleaner for sure! Good catch.